frontend/src/app/[locale]/saved-grants/page.tsx

```diff
@@ -17,9 +22,56 @@ export async function generateMetadata({ params }: LocalizedPageProps) {
   return meta;
 }
 
+const SavedOpportunitiesList = ({
+  opportunities,
+}: {
+  opportunities: Opportunity[];
+}) => {
+  return (
+    <ul className="usa-prose usa-list--unstyled">
+      {opportunities.map((opportunity) => (
+        <li key={opportunity?.opportunity_id}>
+          <SearchResultsListItem opportunity={opportunity} saved={true} />
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const NoSavedOpportunities = ({
+  noSavedCTA,
+  searchButtonText,
+}: {
+  noSavedCTA: React.ReactNode;
+  searchButtonText: string;
+}) => {
+  return (
+    <>
+      <USWDSIcon
+        name="star_outline"
+        className="grid-col-1 usa-icon usa-icon--size-6 margin-top-4"
+      />
+      <div className="margin-top-2 grid-col-11">
+        <p className="usa-intro ">{noSavedCTA}</p>{" "}
+        <Link href="/search">
+          <Button type="button">{searchButtonText}</Button>
+        </Link>
+      </div>
+    </>
+  );
+};
+
 export default async function SavedGrants({ params }: LocalizedPageProps) {
   const { locale } = await params;
   const t = await getTranslations({ locale });
+  const savedOpportunities = await fetchSavedOpportunities();
+  const opportunities = savedOpportunities.map(async (savedOpportunity) => {
```

Comment:
something like "opportunityDetailPromises" would be clearer to me here

Reply:
The `getOpportunityDetails` returns:

```javascript

interface OpportunityApiResponse extends APIResponse {
  data: Opportunity;
}
```

For me it makes sense to describe the type of data we are returning, vs the function used to retrieve it. Does `opportuniyPromises` make more sense?

Reply:
tldr; yep, `opportunityPromises` is great. 

I'm willing to go either way, since the `promise` piece of this is in the type, and explaining the type in the variable name is also generally a no-no, but in cases like this where a synchronous map function returns the promises, and then you await them later, I find it's often confusing at what point we're dealing with unresolved promises and at what point we're dealing with actual data. Somehow making it clear that this variable doesn't yet represent opportunities, but will in the future, seems helpful to me.